bin/generate-podspecs.sh

```diff
@@ -115,7 +117,14 @@ do
     pod=$(basename "$podspec" .podspec)
     path=$(dirname "$podspec")
 
+    # The only one we need is React-jsc, all the others can be skipped.
+    if [[ $pod != 'React-jsc' ]]; then
+      echo "Skipping copying React Native $pod podspec..."
+      continue
+    fi
```

Comment:
I was hesitant about whether to skip the `React-jsc` pod if Hermes is enabled, however, I think this decision could be made by the XCFramework project. Hence, this script will generate all podspecs independently of the engine used (i.e. JSC vs Hermes).

Reply:
+1.

I think it would be handy to skip generating the jsc podspec if we were running the `generate-podspecs.sh` and `bundle exec pod install` as part of the same operation, so that both would read the same "Hermes enabled" config. But given that's not the case, for the moment at least, then "this script will generate all podspecs independently of the engine used (i.e. JSC vs Hermes)" is the way to go.